### PR TITLE
fixed navbar images, profile image and links.

### PR DESCRIPTION
**Pull request by gboyega**
I have fixed the nav bar icons using flex, also added radius and box shadow to the profile image, and also fixed the links as well.

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h2>About Adegboyega</h2>
                 <a href="#"><img src=".\img\GitHub-Mark-32px.png" alt=""></a>
                 <a href=""><img src=".\img\iconfinder_06-facebook_104498.png" alt=""></a>
                 <a href=""> <img src=".\img\iconfinder_Instagram_1298747.png" alt=""></a>
-                <a href=""><img src=".\img\iconfinder_Linkedin_Color_1435161.png" alt=""></a>
+                <a href=""><img src="  .\img/iconfinder_square-linkedin_317725.png" alt=""></a>
```
Comment (GodwinEkuma) [inline About.html]: You are missing an `alt` attribute for the image element.

File: About.html [hunk 2]
```diff
@@ -35,7 +35,7 @@ <h2>About Adegboyega</h2>
                 <a href="#"><img src=".\img\GitHub-Mark-32px.png" alt=""></a>
                 <a href=""><img src=".\img\iconfinder_06-facebook_104498.png" alt=""></a>
                 <a href=""> <img src=".\img\iconfinder_Instagram_1298747.png" alt=""></a>
```
Comment (GodwinEkuma) [inline About.html]: `alt` value should not be empty

File: Contact.html [hunk 3]
```diff
@@ -49,13 +49,13 @@ <h4>Request Call</h4>
...
             <section class="social">
                 <h4>I'm Social</h4>
-                <p><img src="./img/iconfinder_icon-6-mail-envelope-closed_314423.png" alt=""><a href="mailto:#">info@gboyega.com</a></p>
-                <p><img src="./img/iconfinder_phone_3131961.png" alt=""><a href="tel:+2347030884944">+2347032345678</a></p>
-                <p><img src="./img/GitHub-Mark-32px.png" alt=""><a href="#">Github.com/gboyega</a> </p>
-                <p><img src="./img/iconfinder_06-facebook_104498.png" alt=""><a href="#">facebook.com/gboyega</a></p>
-                <p><img src="./img/iconfinder_Instagram_1298747.png" alt="instagram.com/gboyega"><a href="#">instagram.com/gboyega</a></p>
-                <p><img src="./img/iconfinder_Linkedin_Color_1435161.png" alt=""><a href="#">linkedin.com/gboyega</a></p>
-                <p><img src="./img/iconfinder_twitter_294655.png" alt=""><a href="#">twitter.com/gboyega</a></p>
+                <p><img src="./img/iconfinder_icon-6-mail-envelope-closed_314423.png" alt=""><a href="mailto:#"> info@gboyega.com</a></p>
```
Comment (GodwinEkuma) [inline Contact.html]: `alt` value should not be empty

File: Projects.html [hunk 4]
```diff
@@ -60,7 +60,7 @@ <h4>Lrem</h4>
             <a href="#"><img src=".\img\GitHub-Mark-32px.png" alt=""></a>
             <a href=""><img src=".\img\iconfinder_06-facebook_104498.png" alt=""></a>
             <a href=""> <img src=".\img\iconfinder_Instagram_1298747.png" alt=""></a>
-            <a href=""><img src=".\img\iconfinder_Linkedin_Color_1435161.png" alt=""></a>
+            <a href=""><img src="  .\img/iconfinder_square-linkedin_317725.png" alt=""></a>
```
Comment (GodwinEkuma) [inline Projects.html]: also missing an `alt` attribute here

File: About.html (hunk 1 above)
Comment (gboyega) [reply About.html]: Seen. will add alt values and push today.

File: About.html (hunk 2 above)
Comment (gboyega) [reply About.html]: Thank you for noting it. 1 more thing learned.  I will add alt values and push today.

File: Contact.html (hunk 3 above)
Comment (gboyega) [reply Contact.html]: Ok

File: Projects.html (hunk 4 above)
Comment (gboyega) [reply Projects.html]: I did not add alt value in any img tag, i will fix that ASAP

File: About.html (hunk 1 above)
Comment (gboyega) [reply About.html]: I've fixed all alt values now

File: About.html (hunk 2 above)
Comment (gboyega) [reply About.html]: fixed

File: Contact.html (hunk 3 above)
Comment (gboyega) [reply Contact.html]: Fixed

File: Projects.html (hunk 4 above)
Comment (gboyega) [reply Projects.html]: Fixed

**Review by GodwinEkuma — APPROVED** [review]

**Pull request merged**